Description: autopkgtest compat
Author: Xavier Guimard <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2020-11-08

--- a/fast-json-stable-stringify/test/cmp.js
+++ b/fast-json-stable-stringify/test/cmp.js
@@ -1,7 +1,7 @@
 'use strict';
 
 var test = require('tape');
-var stringify = require('../');
+var stringify = require('fast-json-stable-stringify');
 
 test('custom comparison function', function (t) {
     t.plan(1);
--- a/fast-json-stable-stringify/test/nested.js
+++ b/fast-json-stable-stringify/test/nested.js
@@ -1,7 +1,7 @@
 'use strict';
 
 var test = require('tape');
-var stringify = require('../');
+var stringify = require('fast-json-stable-stringify');
 
 test('nested', function (t) {
     t.plan(1);
--- a/fast-json-stable-stringify/test/str.js
+++ b/fast-json-stable-stringify/test/str.js
@@ -1,7 +1,7 @@
 'use strict';
 
 var test = require('tape');
-var stringify = require('../');
+var stringify = require('fast-json-stable-stringify');
 
 test('simple object', function (t) {
     t.plan(1);
--- a/fast-json-stable-stringify/test/to-json.js
+++ b/fast-json-stable-stringify/test/to-json.js
@@ -1,7 +1,7 @@
 'use strict';
 
 var test = require('tape');
-var stringify = require('../');
+var stringify = require('fast-json-stable-stringify');
 
 test('toJSON function', function (t) {
     t.plan(1);
